suites/nautilus/ansible/sanity_ceph_ansible_scale.yaml

```diff
@@ -0,0 +1,200 @@
+      module: test_ansible_roll_over.py
+      config:
+          is_mixed_lvm_configs: True
+          ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_stable_release: nautilus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              journal_size: 1024
+              ceph_stable: True
+              ceph_stable_rh_storage: True
+              fetch_directory: ~/fetch
+              copy_admin_key: true
+              dashboard_enabled: False
+              ceph_conf_overrides:
+                  global:
+                    osd_pool_default_pg_num: 64
+                    osd_default_pool_size: 2
+                    osd_pool_default_pgp_num: 64
+                    mon_max_pg_per_osd: 1024
+                  mon:
+                    mon_allow_pool_delete: true
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "8"
+                - name: "cephfs_metadata"
+                  pgs: "8"
+          add:
+              - node:
+                  node-name: .*node5.*
+                  demon:
+                      - mds
+      desc: add mds daemon
+
+   - test:
+      name: config roll over client daemon
+      polarion-id: CEPH-9581
+      module: test_ansible_roll_over.py
+      config:
+          is_mixed_lvm_configs: True
+          ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_stable_release: nautilus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              journal_size: 1024
+              ceph_stable: True
+              ceph_stable_rh_storage: True
+              fetch_directory: ~/fetch
+              copy_admin_key: true
+              dashboard_enabled: False
+              ceph_conf_overrides:
+                  global:
+                    osd_pool_default_pg_num: 64
+                    osd_default_pool_size: 2
+                    osd_pool_default_pgp_num: 64
+                    mon_max_pg_per_osd: 1024
+                  mon:
+                    mon_allow_pool_delete: true
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "8"
+                - name: "cephfs_metadata"
+                  pgs: "8"
+          add:
+              - node:
+                  node-name: .*node6.*
+                  demon:
+                      - client
+      desc: add client
+
+   - test:
+      name: config roll over nfs daemon
+      polarion-id: CEPH-9581
+      module: test_ansible_roll_over.py
+      config:
+          is_mixed_lvm_configs: True
+          ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_stable_release: nautilus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              journal_size: 1024
```

Comment:
is journal size needed for nautilus ceph cluster?

Reply:
I have made the changes